helper.php

```diff
@@ -125,6 +125,42 @@ public function getTokenEndpoint($service) {
         $service = strtolower($service);
         return $this->getConf($service.'-tokenurl');
     }
+
+    /**
+     * @param bool $string if true returns a nice string for output, otherwise returns array of strings
+     *
+     * @return array|string
+     */
+    public function getValidDomains($string = false) {
+        $validDomains = explode(',', trim($this->getConf('mailRestriction'), ','));
```

Comment:
you might want to do a ``map('trim', $validDomains)`` because users may separate them by ``comma blank``.

I think we introduced an ``array`` type for configs of this kind, if so then ``getConf()`` already returns an array.